### PR TITLE
Assorted internal tweaks

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [x] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [ ] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: cmd/ferretdb/main.go [hunk 1]
```diff
@@ -77,17 +77,21 @@ var cli struct {
...
 	Test struct {
-		RecordsDir string `default:""  help:"Testing flag: directory for record files."`
-		Telemetry  struct {
-			URL            string        `default:"https://beacon.ferretdb.io/" help:"Testing flag: telemetry URL."`
-			UndecidedDelay time.Duration `default:"1h"                          help:"Testing flag: delay for undecided state."`
-			ReportInterval time.Duration `default:"24h"                         help:"Testing flag: report interval."`
-			ReportTimeout  time.Duration `default:"5s"                          help:"Testing flag: report timeout."`
+		RecordsDir string `default:"" help:"Testing flag: directory for record files."`
+
+		// TODO https://github.com/FerretDB/FerretDB/issues/1912
```
Comment (chilagrow) [inline cmd/ferretdb/main.go:82]: Great point!

**Review by chilagrow — APPROVED** [review]

File: .github/RELEASE_CHECKLIST.md [hunk 2]
```diff
@@ -37,5 +37,10 @@
...
 4. Announce it on Slack.
-5. Bump the latest version on [Beacon](https://beacon.ferretdb.io).
-6. Publish blog post, etc.
+
+## Soon after
+
+1. Bump the latest version on [Beacon](https://beacon.ferretdb.io).
+2. Publish and announce blog post.
+3. Tweet, toot.
+4. Update NixOS package.
```
Comment (rumyantseva) [inline .github/RELEASE_CHECKLIST.md:46]: That would be great also to have documentation things like "Bump the latest version on Beacon" and "Update NixOS package", but as far as I understand, it's a checklist at the moment.

File: integration/setup/common.go [hunk 3]
```diff
@@ -60,6 +60,9 @@ var (
 
 	recordsDirF = flag.String("records-dir", "", "directory for record files")
 
+	// TODO https://github.com/FerretDB/FerretDB/issues/1912
+	_ = flag.Bool("disable-pushdown", false, "disable query pushdown")
```
Comment (rumyantseva) [inline integration/setup/common.go:64]: 👍 (cc @noisersup, yay, we will have this flag!)

**Review by rumyantseva — APPROVED** [review]
LGTM

File: .github/RELEASE_CHECKLIST.md (hunk 2 above)
Comment (AlekSi) [reply .github/RELEASE_CHECKLIST.md:46]: It is also more-or-less a personal checklist — most things could be done only by me, but that’s changing slowly

**Pull request merged**